Update 13030_generic_vtol_quad_tiltrotor

dhruvilodonata opened the pull request:
Added line 43 and 46, As when the CS2 and CS3 is changed to V tail left and right, the CA_SV_CS2_TRQ_Y and CA_SV_CS2_TRQ_P is not available in QGC.  adding these two parameters would help to use the same frame with different Tail type.

<!--

Thank you for your contribution!

Get early feedback through
- Dronecode Discord: https://discord.gg/dronecode
- PX4 Discuss: http://discuss.px4.io/
- opening a draft pr and sharing the link

-->

### Solved Problem
When ... I found that ...

Fixes #{Github issue ID}

### Solution
- Add ... for ...
- Refactor ...

### Changelog Entry
For release notes:
```
Feature/Bugfix XYZ
New parameter: XYZ_Z
Documentation: Need to clarify page ... / done, read docs.px4.io/...
```

### Alternatives
We could also ...

### Test coverage
- Unit/integration test: ...
- Simulation/hardware testing logs: https://review.px4.io/

### Context
Related links, screenshot before/after, video

sfuhrer reviewed (changes requested):
Sorry I don't understand what your motivation for this change is. The generic VTOL has a T-tail, not a V-tail.

dhruvilodonata commented:
I changed the surface type to V tail left and V tail right, for CS2 and CS3 respectively. The problem I am facing right now is, in generic model, CS2 is Elevator and when i changed it to V tail left, there is no  CA_SV_CS2_TRQ_Y which assigns half yaw value to CS2 and with CS3 to V tail right, there is no CA_SV_CS2_TRQ_P which assigns half pitch value to CS3. thats why even after changing the CS2 and CS3 i am only getting pitch and yaw output to the servo. Ideally i should receive yaw and pitch both output from the same servo

dhruvilodonata commented:
I am using  PX4 1.13 and QGC 2.4.6. When i change the CS2 and CS3 parameters to V tail left and right, the parameters   CA_SV_CS2_TRQ_Y and  CA_SV_CS3_TRQ_P is not available in parameter list in QGC. And default value of these two parameters are 0 as CS2 is elevator and CS3 is Rudder. After changing CS2 and CS3 to V tail, the default value is not changing to 0.5 and 0.5.

dhruvilodonata commented:
@sfuhrer, could you please check and merge these two paramerters in the main code.

sfuhrer commented:
So you're saying that for you it looks different than [here](https://youtu.be/NQwepGLMzZ4?si=4ZUj7R8_Tmf7sSBv&t=655)? Have you tried with 1.14 or latest main branch?
